metis.md

```diff
@@ -144,3 +144,31 @@ If we are dissatisfied with our progress we may cancel the upload:
 ```
 { action: 'cancel' }
 ```
+
+## Setup
+
+### Installation
+
+When you update or install the project, make sure to run migrations. You should do this with the built-in command provided in `$ bin/metis migrate` instead of directly using `Sequel`.
+
+To run migrations against your test database, run `$ METIS_ENV=test bin/metis migrate`.
+
+### Data Directory
+
+You can configure where the data blocks are stored in your `config.yml` file. The environment (`test`, `development`, etc.) should have a `:data_path:` value set. Make sure to also create a sub-directory `data_blocks`. For example, if your `config.yml` looks like:
+
+```yml
+:development:
+  :data_path: ./data
+```
+
+On your disk you'll want to create the following directories:
+
+```sh
+data/
+  data_blocks/
+```
```

Comment:
Do you also need to create an `uploads` directory or does Metis do that for you?

Reply:
Good point, I had to do it manually, even though the folder winds up empty.